getting-started/installation/general-installation/installation-community-containerized.markdown

```diff
@@ -0,0 +1,249 @@
+---
+layout: default
+title: Installing Community
```

Comment:
This title does not reflect what this tutorial is about.

Reply:
Perhaps
```suggestion
title: Installing CFEngine community in containerized environment
```

Most titles are sentence cased. Though I have some weird affinity for long specific names so maybe there is something shorter.

Reply:
Apologies, it was a copy-paste remnant from https://github.com/cfengine/documentation/blob/master/getting-started/installation/general-installation/installation-community.markdown?plain=1#L3 😁
I'll update it to 
> Installing Community Using Containers
To look similar with the other titles.